pvlib/iotools/pvgis.py

```diff
@@ -0,0 +1,156 @@
+"""
+Get, read, and parse data from `PVGIS <https://ec.europa.eu/jrc/en/pvgis>`_.
+
+For more information, see the following links:
+* `Interactive Tools <https://re.jrc.ec.europa.eu/pvg_tools/en/tools.html>`_
+* `Data downloads <https://ec.europa.eu/jrc/en/PVGIS/downloads/data>`_
+* `User manual docs <https://ec.europa.eu/jrc/en/PVGIS/docs/usermanual>`_
+
+More detailed information about the API for TMY and hourly radiation are here:
+* `TMY <https://ec.europa.eu/jrc/en/PVGIS/tools/tmy>`_
+* `hourly radiation
+  <https://ec.europa.eu/jrc/en/PVGIS/tools/hourly-radiation>`_
+* `daily radiation <https://ec.europa.eu/jrc/en/PVGIS/tools/daily-radiation>`_
+* `monthly radiation
+  <https://ec.europa.eu/jrc/en/PVGIS/tools/monthly-radiation>`_
+"""
+import io
+import tempfile
+import requests
+import pandas as pd
+from pvlib.iotools import read_epw
+
+URL = 'https://re.jrc.ec.europa.eu/api/'
+
+
+def get_pvgis_tmy(lat, lon, outputformat='json', usehorizon=True,
```

Comment:
A suggestion : add the url as a kwarg defaulting to the constant, in case they change the server.